src/test/miner_tests.cpp

```diff
@@ -39,6 +39,8 @@ class HasReason {
 
 static CFeeRate blockMinFeeRate = CFeeRate(DEFAULT_BLOCK_MIN_TX_FEE);
 
+static CChain& nonConstChainActive = const_cast<CChain&>(chainActive);
```

Comment:
In commit "Add a const_cast'ed reference to chainActive in miner_tests",

Why add an unused variable? Could add just when it's needed?

Answer: Because later it's used in "scripted-diff: Use nonCosntChainActive in miner_tests" commit.